docs/eventing/sources/pingsource.md

```diff
@@ -1,196 +1,261 @@
 ---
-title: "PingSource"
+title: "PingSource example"
```

Comment:
I'm not sure about the heading, is this just an example topic? Where would context info about the source itself be if it's not here?

Reply:
Yep, I'll remove it.